allspice/apiobject.py

```diff
@@ -1835,6 +1835,21 @@ def commit(self):
         }
         self._commit(args)
 
+    def download(self) -> bytes:
+        """
+        Download the raw, binary data of this asset.
+
+        Note: if the file you are requesting is a text file, you might want to
+        use .decode() on the result to get a string. For example:
+
+            asset.download().decode("utf-8")
+        """
+
+        return self.allspice_client.requests.get(
+            self.browser_download_url,
```

Comment:
Where is this defined? I don't see it, so I'm guessing it's computed dynamically somewhere. Can you point me to where?

Reply:
Never mind. I found it.